integration-tests/api-gateway/aws/crypto/client.ts

```diff
@@ -52,7 +60,21 @@ export const buildJarAuthorizationRequest = async (params: any) => {
   };
 };
 
-const uuid = () => {
+
+export const buildPrivateKeyJwtParams = async (code: string, params: any, signingKey: JWK) => {
+  const signedJwt = await signJwtForToken(params, signingKey);
+
+  return new URLSearchParams({
+    client_assertion_type: "urn:ietf:params:oauth:client-assertion-type:jwt-bearer",
+    code: code,
+    grant_type: "authorization_code",
+    redirect_uri: "https://cri.core.stubs.account.gov.uk/callback",
```

Comment:
can this be moved to the a const file, maybe `test-data` for now be am open for a better name